includes/drush.api.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Executes a deploy function which is intended to update data, like entities,
+ * after config is imported during a deployment.
+ *
+ * These are a higher level alternative to hook_update_n and hook_post_update_NAME
+ * functions and have to be placed in a MODULE.deploy.php file.
+ *
+ * NAME can be arbitrary machine names. In contrast to hook_update_N() the
+ * alphanumeric naming of functions in the file is the only thing which ensures
+ * the execution order of those functions. If update order is mandatory,
+ * you should add numerical prefix to NAME or make it completely numerical.
+ *
+ * Drupal also ensures to not execute the same hook_post_update_NAME() function
+ * twice.
+ *
+ * @section sec_bulk Batch updates
+ * If running your update all at once could possibly cause PHP to time out, use
```

Comment:
This section is a bit confusing because Drush always executes these at the CLI in one process. There is no way at present to execute them via the web and use batches. I wonder if we should remove it.

Reply:
Do you mean remove the whole batch functionality from deploy hooks? Or just the documentation? Maybe we commit this documentation for now and remove it again if we end up removing batch support from deploy hooks.